ts/twitter/src/app/(protected)/message/_components/messages.tsx

```diff
@@ -0,0 +1,128 @@
+"use client";
+
+import { BackIcon, CreateMailIcon, SettingsIcon } from "@/lib/components/icons";
+import { useColorModeValue } from "@/lib/components/ui/color-mode";
+import { Tooltip } from "@/lib/components/ui/tooltip";
+import {
+  Box,
+  Button,
+  Link as ChakraLink,
+  Flex,
+  IconButton,
+  Spacer,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+import NextLink from "next/link";
+import type React from "react";
+
+export const Messages: React.FC = () => {
+  return (
+    <VStack align="stretch">
+      <Box divideY="1px">
+        <Box width="100%" padding="0">
+          <Flex mx="4px" gap="4" align="center">
+            <Tooltip
+              content="Back"
+              contentProps={{
+                style: {
+                  backgroundColor: useColorModeValue("gray.800", "gray"),
+                  color: "white",
+                },
+              }}
+            >
+              <ChakraLink asChild>
+                <NextLink href="/home" scroll={false}>
+                  <IconButton
+                    my="5px"
+                    bg="transparent"
+                    borderRadius="full"
+                    color={useColorModeValue("black", "white")}
+                    _hover={{ bg: useColorModeValue("blue.100", "gray.900") }}
+                  >
+                    <BackIcon />
+                  </IconButton>
+                </NextLink>
+              </ChakraLink>
+            </Tooltip>
+            <Text fontSize="xl" fontWeight="bold">
+              Messages
+            </Text>
+            <Spacer />
+            <Flex mx="4px" gap="0" align="center">
+              <IconButton
+                my="5px"
+                bg="transparent"
+                borderRadius="full"
+                color={useColorModeValue("black", "white")}
+                _hover={{ bg: useColorModeValue("blue.100", "gray.900") }}
+              >
+                <SettingsIcon />
+              </IconButton>
+              <IconButton
+                my="5px"
+                bg="transparent"
+                borderRadius="full"
+                color={useColorModeValue("black", "white")}
+                _hover={{ bg: useColorModeValue("blue.100", "gray.900") }}
+              >
+                <CreateMailIcon />
+              </IconButton>
+            </Flex>
+          </Flex>
+        </Box>
+        <Box
+          minHeight="600px"
+          display="flex"
+          alignItems="center"
+          justifyContent="center"
+        >
+          <VStack gap={6} maxW="400px" px={4}>
```

Comment:
Curly braces should only be used when the value is dynamic or a JavaScript expression.
```suggestion
          <VStack gap="6" maxW="400px" px="4">
```